### PR TITLE
Update 2.0 release notes, table formatting, organize, remove 'see full list'

### PR DESCRIPTION
**Pull request by DanVanAtta**

### CODE REVIEW

File: css/main.scss [hunk 1]
```diff
@@ -16,6 +16,24 @@ $grey-color: #828282;
...
+table {
+	border-collapse: collapse;
+	width: 100%;
+	text-align: left;
+	border: 1px solid #c6c9cc;
+	border-radius: 5px;
+ 	padding: 10px 20px;
+	th {
+    text-align: center;
+  }
```
Comment (RoiEXLab) [inline css/main.scss:28]: The formatting is a bit odd

Comment (DanVanAtta) [reply css/main.scss:28]: even for scss? I ran this into a css->scss converter, kinda trusting the output. The th block could use indentation though.

Comment (RoiEXLab) [reply css/main.scss:28]: I'm just talking about the closing curly brace and the misaligned "text-align" property here

File: css/main.scss [hunk 2]
```diff
@@ -16,6 +16,24 @@ $grey-color: #828282;
...
+  td {
+		border-color: #c6c9cc;
+		border-style: solid;
+		border-width: 1px 1px 1px 1px;
+  	padding: 5px 10px;
```
Comment (RoiEXLab) [inline css/main.scss:33]: Also this property is a bit odd

**Pull request merged**